mobile/asa-go/src/components/map/ASAGoMap.tsx

```diff
@@ -230,6 +231,7 @@ const ASAGoMap = ({
         selectedFireShape
       )
     );
+    centerOnFireShape(map, selectedFireShape, fireZoneExtentsMap, popup);
```

Comment:
I don't think we should be panning the map when a user taps on it to select a fire shape, that would be unexpected in my opinion. But I think centering the map when a user selected a fire shape from a non-map tab makes sense.